provisioning/stream/kubernetes/templates/api/deployment.yaml

```diff
@@ -39,44 +39,41 @@ spec:
             }
           }
     spec:
-      terminationGracePeriodSeconds: 600
+      terminationGracePeriodSeconds: 60
       containers:
-        - name: buffer-api
-          image: $BUFFER_API_REPOSITORY:$BUFFER_API_IMAGE_TAG
+        - name: stream-api
+          image: $STREAM_IMAGE_REPOSITORY:$STREAM_IMAGE_TAG
+          command: [ "/app/service" ]
+          args: [ "--config-file", "/stream/config/config.yaml", "api" ] # run API component
           resources:
             requests:
               cpu: "200m"
```

Comment:
není 200 zbytečně málo? dal bych 500 a určitě pak i limit aby kontejner byl v rámci QoS guaranteed, dejme třeba 1000m

Reply:
:man_shrugging: naposledy, ked som to riesil s Kacurezom, tak chcel vsade `200m`, bez ohladu na to, co to robi, upravujem.

Reply:
Fixnute: https://github.com/keboola/keboola-as-code/pull/1860/commits/14aa80405fcbb78822da1ad391e32e8c1837bc68

Reply:
ty cpu jsou tedy možná moc, nechcme to pak a poladíme to až budeme v produkci https://keboolaglobal.slack.com/archives/C055K9CRFDX/p1719823809995919